decidim-core/app/controllers/concerns/decidim/amendments_controller.rb

```diff
@@ -42,13 +42,40 @@ def create
       end
     end
 
-    private
-
     def reject; end
 
-    def review; end
+    def review
+      emendation_fields_form = emendation.form.from_model(emendation)
+      params = emendation.attributes
+      params[:id] = emendation.amendment.id
+
+      @form = Decidim::Amendable::ReviewForm.from_params(
```

Comment:
Maybe add `from_model` to `ReviewForm` and move the logic there is a better option.

Reply:
Thanks very much. I've been tinkering with the code and now I understand it better. It's much nicer the way you suggest and I think I'll be able to do the proper validations there and dispense with the validate command. I'll refactor it soon.